models/gc_grt123_lung_cancer/meta.json

```diff
@@ -0,0 +1,144 @@
+        "uri": "https://arxiv.org/pdf/1711.08324.pdf"
+      },
+      {
+        "title": "Deep Learning for Lung Cancer Detection on Screening CT Scans: Results of a Large-Scale Public Competition and an Observer Study with 11 Radiologists",
+        "uri": "https://pubmed.ncbi.nlm.nih.gov/34870218/"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/bodyct-dsb2017-grt123",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk.",
+      "references": [
+        {
+          "label": "Lung cancer risk estimation algorithm on grand-challenge",
+          "uri": "https://grand-challenge.org/algorithms/dsb2017-grt123/"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "The nodule detection was evaluated on the validation set of Data Science Bowl 2017 challenge. It contains data from 198 cases and there are 71 (7 nodules smaller than 6 mm are ruled out) nodules in total. The Free Response Operating Characteristic (FROC) is used to evaluate the performance and can be viewed in the publication. The Case classification is evaluated using the Area Under the Curve (AUC) metric.",
+      "references": [{
+        "label": "Data Science Bowl 2017 challenge",
+        "uri": "https://www.kaggle.com/c/data-science-bowl-2017"
+      }],
+      "tables": [{
+        "label": "AUC for the Case classification",
+        "entries": {
+          "AUC training set": "0.90",
+          "AUC test set": "0.87"
+        }
+      }]
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The model was evaluated against a private dataset of 300 low-dose CT images, containing 100 cancer-positive scans and 200 cancer-negative scans.",
+      "references": [
+        {
+          "label": "Evaluation paper",
+          "uri": "https://pubmed.ncbi.nlm.nih.gov/34870218/"
+        }
+      ],
+      "tables": []
+    },
+    "training": {
+      "title": "Training data",
+      "text": "The Luna dataset includes the images from the LIDC/IDRI dataset in a different format, together with additional annotations. The LIDC/IDRI dataset is available at https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI under a Creative Commons Attribution 3.0 Unported License.",
+      "references": [
+        {
+          "label": "LUng Nodule Analysis 2016 dataset part 1",
+          "uri": "https://zenodo.org/record/3723295"
+        },
+        {
+          "label": "LUng Nodule Analysis 2016 dataset part 2",
+          "uri": "https://zenodo.org/record/4121926"
+        },
+        {
+          "label": "Data Science Bowl 2017 dataset",
+          "uri": "https://www.kaggle.com/competitions/data-science-bowl-2017/data"
```

Comment:
Unclear how this link is relevant. The data has been removed from kaggle.

Reply:
See comment above